solutions/io.lionweb.mps.cmdline/readme.md

```diff
@@ -75,20 +76,35 @@ Contents of `config-file.json`:
 
 #### Test exports
 
-The directory `test-project/` can be used to test the command-line exporter.
-Run
+The directories `test-project/` and `test-project-externalLib` contain MPS projects that can be used to test the command-line exporter.
+
+Within `test-project/`, run
+
+```shell
+source scripts/export-library.sh
+source scripts/export-multiple.sh
+source scripts/export-foo.sh
+```
+
+(or a similar incantation suitable for your OS and shell) to export three example languages, and compare the exports to the reference exports in `expected/`.
+The languages' exports end up in the `actual/` directory.
+The `FooLanguage` language resides within the project, but the `library` and `multi` language are bundled.
+The `export-multiple.sh` script also demonstrates the use of a JSON configuration file — in this case [`export-multiple.config.json`](../../test-project/export-multiple.config.json).
+
+Within `test-project-externalLib`, run
```

Comment:
Should we mention that (and how) we first need to run gradle to download the external dependencies -- before opening the project in MPS?